docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+          width: '180'
+        }"
+        :items="deepClone(defaultItemsUnselected)"
+      >
+        <template #default>
+          <KButton
+            appearance="secondary"
+            size="small"
+            class="non-visual-button"
+          >
+            <template #icon>
+              <KIcon
+                icon="more"
+                color="var(--black-400)"
+                size="16"
+              />
+            </template>
+          </KButton>
+        </template>
+      </KDropdownMenu>
+    </template>
+  </KCard>
+</div>
+
+```html
+<KCard
+  title="Card Title"
+  body="Body Content"
+>
+  <template #actions>
+    <KDropdownMenu
+      :kpop-attributes="{
+        popoverClasses: 'mt-5',
+        width: '180'
+      }"
+      :items="deepClone(defaultItemsUnselected)"
+    >
+      <template #default>
+        <KButton
+          appearance="secondary"
+          size="small"
+          class="non-visual-button"
+        >
+          <template #icon>
+            <KIcon
+              icon="more"
+              color="var(--black-400)"
+              size="16"
+            />
+          </template>
+        </KButton>
+      </template>
+    </KDropdownMenu>
+  </template>
+</KCard>
+```
+
+### disabled
+
+Use this prop to disable the dropdown, can be used in conjunction with `disabledTooltip` prop.
+
+### disabledTooltip
+
+Text to display on hover if dropdown is disabled.
+
+<div>
+  <KDropdownMenu
+    label="Documentation"
+    :disabled="true"
+    disabled-tooltip="You can't click me"
+    :items="deepClone(defaultItemsUnselected)"
+  />
+</div>
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :disabled="true"
+  disabled-tooltip="You can't click me"
+  :items="deepClone(defaultItemsUnselected)"
+/>
+```
+
+## Slots
+
+There are 2 supported slots:
+
+- `default` - The trigger element for opening/closing the menu. Returns the `isOpen` state.
+- `items` - For an example of using the items slot see the [`KDropdownItem`](#KDropdownItem) section.
+
+<div>
+  <KDropdownMenu :items="deepClone(defaultItemsUnselected)">
+    <template #default="{ isOpen }">
+        <KButton
+          :is-open="isOpen"
+          appearance="creation"
+        >
+          Menu
+        </KButton>
+      </template>
+  </KDropdownMenu>
+</div>
+
+```html
+<KDropdownMenu :items="items">
+  <template #default="{ isOpen }">
+      <KButton
+        :is-open="isOpen"
+        appearance="creation"
+      >
+        Menu
+      </KButton>
+    </template>
+</KDropdownMenu>
+```
+
+## KDropdownItem
+
+**KDropdownMenu** generates a **KDropdownItem** for each item in the `items` property. At the most basic level, **KDropdownItem** is a wrapper around each item to display it correctly inside `KDropdownMenu`. You can use the `items` slot of the `KDropdownMenu` to manually create your own menu items.
+
+### Properties
+
+- `item` - the properties the router-link is built from, it expects a `label` and a `to`.
```

Comment:
See my comment about router-links -- ideally I think we should require using the slot for router-links so that we don't have a Vue router peer dependency